tests/acceptance/run.sh

```diff
@@ -96,6 +96,11 @@ if [ -n "${EXPECTED_FAILURES_FILE}" ]; then
       continue
     fi
 
+    if [[ "${LINE}" =~ -\ \[(.*?)] ]]; then
+          LINE="${BASH_REMATCH[1]}"
+        else
+          echo "Incorrect format for ${LINE}"
```

Comment:
Note: we can also be flexible and let people write free text paragraphs in the expected-failures `.md` file, as well as headings that start with `#`. So here we can just ignore such lines:
```suggestion
          continue
```